hal/halx86/legacy/arcmap.c

```diff
@@ -28,13 +35,12 @@ typedef struct _PNP_DETECT_IDENTIFIER_MAP
     ULONG Counter;
 } PNP_DETECT_IDENTIFIER_MAP;
 
+extern LIST_ENTRY HalpPdoListHead;
+
 /* DATA **********************************************************************/
 
 static UNICODE_STRING IdentifierU = RTL_CONSTANT_STRING(L"Identifier");
-static UNICODE_STRING HardwareIDU = RTL_CONSTANT_STRING(L"HardwareID");
 static UNICODE_STRING ConfigurationDataU = RTL_CONSTANT_STRING(L"Configuration Data");
-static UNICODE_STRING BootConfigU = RTL_CONSTANT_STRING(L"BootConfig");
-static UNICODE_STRING LogConfU = RTL_CONSTANT_STRING(L"LogConf");
```

Comment:
Which module is going to deal with those values now?

Reply:
These registry keys are written by the PnP manager